test/verify/check-services

```diff
@@ -470,7 +470,7 @@ WantedBy=default.target
         b.click("#timer-save-button")
         b.wait_popdown("timer-dialog")
         b.wait_present(svc_sel('boot_timer.timer'))
-        m.execute("systemctl enable boot_timer")
+        m.execute("systemctl enable boot_timer.timer")
```

Comment:
I don't understand why the *test* would enable the timer? If you create a timer in the Cockpit web UI, wouldn't we expect it to get enabled right away?